Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -925,17 +897,9 @@ BroadcastReceivers should use the `android:permission` attribute;  otherwise, ot
 
 ### Dynamic Analysis
 
-You can enumerate IPC components with Drozer. To list all exported IPC components, use the module `app.package.attacksurface`:
+You can enumerate IPC components with [MobSF](0x08-Testing-Tools.md#mobsf "MobSF"). To list all exported IPC components, upload the APK file and the components collection will be displayed in the following screen:
 
-```bash
-dz> run app.package.attacksurface com.mwr.example.sieve
-Attack Surface:
-  3 activities exported
-  0 broadcast receivers exported
-  2 content providers exported
-  2 services exported
-    is debuggable
-```
+<img src="Images/Chapters/0x05h/MobSF_Show_Components.png" alt="MobSF components display" />
 
 #### Content Providers
 
```

Comment:
please consider also replacing the commands below that are also for drozer. For listing the content providers you could add the alternative of parsing the AndroidManifest. For querying:

- via adb:
```
adb shell content insert --uri <URI> [--user <USER_ID>] --bind <BINDING> [--bind <BINDING>...]
```
- or via Frida: https://codeshare.frida.re/@leolashkevych/android-query-provider/

maybe you know some more?

Reply:
Final status of my work this morning:
* **Activities:** Drozer replaced
* **Content Providers:** Drozer replaced
* **Broadcast Receivers:** Drozer replaced
* **Services:** Drozer not replaced because I did not found a acceptable replacement.
* **Sniffing Intents:** Drozer not replaced because I did not found a acceptable replacement but additional ADB command added.